src/pages/RepoPage/FailedTestsTab/FailedTestsPage/MetricsSection/MetricsSection.tsx

```diff
@@ -192,7 +239,16 @@ const TotalFailuresCard = ({
         </MetricCard.Title>
       </MetricCard.Header>
       <MetricCard.Content>
-        {totalFails}
+        <button
+          className={cs('text-ds-blue-default hover:underline', {
+            'font-semibold': isSelected,
+          })}
+          onClick={() => {
+            updateParams({ parameter: 'FAILED_TESTS' })
```

Comment:
Pulling these out to an enum could be helpful

Reply:
I thought about it, I think the team has mixed feelings about enums and we haven't really come to a conclusion for when to use vs. not. I'll leave for now but will bring up in our next team meeting and update before launch if needed :)